msal/src/Microsoft.Identity.Client/IAccount.cs

```diff
@@ -53,7 +53,7 @@ public interface IAccount
         /// <summary>
         /// AccountId of the home account for the user. This uniquely identifies the user across AAD tenants. 
         /// </summary>
-        /// <remarks>Can be null</remarks>
+        /// <remarks>Can be null, for example with users migrating from ADAL v3</remarks>
```

Comment:
Can be null, for example if this account was migrated to MSAL.NET from ADAL.NET v3's token cache